provider/datagen/src/transform/cldr/datetime/mod.rs

```diff
@@ -346,45 +346,15 @@ impl_data_provider!(
 );
 impl_data_provider!(IndianDateSymbolsV1Marker, symbols::convert_dates, "indian");
 impl_data_provider!(
-    IslamicCivilDateLengthsV1Marker,
+    IslamicDateLengthsV1Marker,
     |dates, _| DateLengthsV1::from(dates),
     "islamicc"
```

Comment:
What is this used for?

Reply:
The source cldr calendar

Reply:
So CLDR does have different data for the different calendars?

Reply:
*separate data

Reply:
Yes but it's identical (and should be identical), we can potentially check it.

Reply:
That's what I'm trying to get at. If it doesn't matter which CLDR calendar we use, we should verify that they're all the same.